### PR TITLE
Add detailed installation instructions

### PR DESCRIPTION
**Pull request by mpwp**
Removed broken links at the top.
Added some cabal commands.

### CODE REVIEW

**Comment by mpwp** [comment]
Closes #902

File: README.md [hunk 1]
```diff
@@ -4,28 +4,49 @@
...
+Now, it's recommended to use a static configuration which makes it possible to distribute binaries without the entire Haskell ecosystem.
+
+*Dynamic configuration is still available using a [separate package](https://hackage.haskell.org/package/yi-dynamic-configuration).*
+
+See this [article](https://yi-editor.github.io/posts/2017-01-06-dyre/) for more detailed explanations about static and dynamic configuration.
+
+### Static example configurations
+The static example configurations don't need to be placed in any specific directory (but you can still use `~/.config/yi`).
+   1. Copy the example configuration (only the contents of the folder) to your `yi` folder.
+   2. Edit *stack.yaml* file so that `location: ` would point to the root of the Yi source code.
```
Comment (ethercrow) [inline README.md]: This is only necessary if you want to build against a local copy of Yi source code. If you're fine with vanilla packages, delete the `location:` section instead. Basically, if you are not working on Yi itself at the moment, you don't need to refer to local copy (or even have it on your disk).

Comment (mpwp) [inline README.md]: Sure, step 2 deleted.

**Comment by ethercrow** [comment]
Great, thank you!

**Pull request merged**